src/widget/wsearchlineedit.cpp

```diff
@@ -62,9 +62,7 @@ WSearchLineEdit::WSearchLineEdit(QWidget* pParent)
 
     setAcceptDrops(false);
 
-    QPixmap pixmap(":/images/library/ic_library_cross.svg");
-    m_clearButton->setIcon(QIcon(pixmap));
-    m_clearButton->setIconSize(pixmap.size());
+    m_clearButton->setObjectName("pushButtonClearSearch");
```

Comment:
We can get rid of the string compare if we add a function to the WSearchLineEdit 
hasClearButtonFocus.

Reply:
yes, we could do the same for dlgAutoDJ and dlgRecording.
locally I managed to filter all relevant buttons (AutoDJ & Recording)